src/App.vue

```diff
@@ -273,20 +273,12 @@ export default class App extends Mixins(
 
   /** Whether the old webchat should be enabled. */
   get enableOldWebchat (): boolean {
-    if (getFeatureFlag('enable-web-chat')) {
-      return !!getFeatureFlag('enable-web-chat')
-    } else {
-      return false // Failsafe value
-    }
+    return getFeatureFlag('enable-web-chat')
```

Comment:
`getFeatureFlag()` returns a type of any. If you use `!!` then you get a boolean instead.

Reply:
You're right! I was following like this here: https://github.com/bcgov/namerequest/blob/e279a02b010ffaafe8092b69267a070dddb3c8fa/src/components/common/applicant-info-2.vue#L280
I'll fix it.

Reply:
Hah, that other code should be fixed :)

(I think it evaluates the same, but typing is helpful to prevent errors early on.)

Reply:
Got it. Shall I just add the `!!` since I'm already on the subject of feature flags or leave it for another ticket?

Reply:
Sure, fix it.